### PR TITLE
[ci] push docker images to ecr

### PR DESCRIPTION
**Pull request by rexhoffman**
## Motivation

Push dockerhub images to novi ecr with current ecr locations on build in release/test branch.

Tested here: https://app.circleci.com/pipelines/github/libra/libra/26908/workflows/15ba488a-5d46-4941-9f7b-992687c11bfa

Images in ECR have names like:

853397791086.dkr.ecr.us-west-2.amazonaws.com/libra_init:test-1_24e72d69

### Have you read the [Contributing Guidelines on pull requests]

Yes

## Test Plan

Tested here:
https://app.circleci.com/pipelines/github/libra/libra/26908/workflows/15ba488a-5d46-4941-9f7b-992687c11bfa

## Related PRs

None

### CODE REVIEW

**Comment by mgorven** [comment]
Thanks :-)

File: .circleci/config.yml [hunk 1]
```diff
@@ -618,20 +633,19 @@ jobs:
             curl -o /tmp/pr https://api.github.com/repos/libra/libra/pulls/${pr_num}
             export BRANCH=$( cat /tmp/pr | jq ".base .ref" | sed 's/"//g' )
 
-            export IS_RELEASE=$( if [[ "$BRANCH" =~ "^v[0-9|.]+-release$" ]]; then echo true; else echo false; fi )
-            export IS_TEST=$( if [[ "$BRANCH" =~ "^testflow[0-9|.]+$" ]]; then echo true; else echo false; fi )
-            export IS_PRE_RELEASE=$( if [[ "$BRANCH" =~ "^premainnet-[0-9|.]+$" ]]; then echo true; else echo false; fi )
+            export IS_RELEASE=$( if [[ "$BRANCH" =~ "^release-[0-9|.]+$" ]]; then echo true; else echo false; fi )
```
Comment (sausagee) [inline .circleci/config.yml:636]: Are we changing the release branch from `v1-release` to `release-1`?

Comment (rexhoffman) [reply .circleci/config.yml:636]: @bmwill preferred this naming structure.   And it's easier for humans, so I was fine with changing to this.   Guess I need to track down the docs :D.

Comment (bmwill) [reply .circleci/config.yml:636]: I believe I said it was easier to read, but that it wasn't important or an issue either way. You jumped on changing it :)

**Pull request closed without merging**